qcodes/instrument/parameter.py

```diff
@@ -568,6 +604,16 @@ def validate(self, value: ParamDataType) -> None:
 
     @property
     def step(self) -> Optional[Number]:
+        """
+        Stepsize that this Parameter uses during set operation.
+        If step is a positive number, this is the maximum value change
```

Comment:
What if step is a negative number? Do we raise?

Reply:
This is documented below in the nice docstring of the setter, which unfortunately is never used in the rendered docs 😛